classes/Pods.php

```diff
@@ -1007,82 +1007,15 @@ public function field( $name, $single = null, $raw = false ) {
 				}
 			}
 
-			if ( 'post_type' === $this->pod_data['type'] && ! isset( $this->fields[ $params->name ] ) ) {
-				if ( ! isset( $this->fields['post_thumbnail'] ) && ( 'post_thumbnail' === $params->name || 0 === strpos( $params->name, 'post_thumbnail.' ) ) ) {
-					$size = 'thumbnail';
-
-					if ( 0 === strpos( $params->name, 'post_thumbnail.' ) ) {
-						$field_names = explode( '.', $params->name );
-
-						if ( isset( $field_names[1] ) ) {
-							$size = $field_names[1];
-						}
-					}
-
-					// Pods will auto-get the thumbnail ID if this isn't an attachment.
-					$value = pods_image( $this->id(), $size, 0, null, true );
-
-					$object_field_found = true;
-				} elseif ( ! isset( $this->fields['post_thumbnail_url'] ) && ( 'post_thumbnail_url' === $params->name || 0 === strpos( $params->name, 'post_thumbnail_url.' ) ) ) {
-					$size = 'thumbnail';
-
-					if ( 0 === strpos( $params->name, 'post_thumbnail_url.' ) ) {
-						$field_names = explode( '.', $params->name );
-
-						if ( isset( $field_names[1] ) ) {
-							$size = $field_names[1];
-						}
-					}
-
-					// Pods will auto-get the thumbnail ID if this isn't an attachment.
-					$value = pods_image_url( $this->id(), $size, 0, true );
-
-					$object_field_found = true;
-				} elseif ( 0 === strpos( $params->name, 'image_attachment.' ) ) {
-					$size = 'thumbnail';
-
-					$image_id = 0;
-
-					$field_names = explode( '.', $params->name );
-
-					if ( isset( $field_names[1] ) ) {
-						$image_id = $field_names[1];
-					}
-
-					if ( isset( $field_names[2] ) ) {
-						$size = $field_names[2];
-					}
-
-					if ( ! empty( $image_id ) ) {
-						$value = pods_image( $image_id, $size, 0, null, true );
-
-						if ( ! empty( $value ) ) {
-							$object_field_found = true;
-						}
-					}
-				} elseif ( 0 === strpos( $params->name, 'image_attachment_url.' ) ) {
-					$size = 'thumbnail';
-
-					$image_id = 0;
-
-					$field_names = explode( '.', $params->name );
-
-					if ( isset( $field_names[1] ) ) {
-						$image_id = $field_names[1];
-					}
-
-					if ( isset( $field_names[2] ) ) {
-						$size = $field_names[2];
-					}
-
-					if ( ! empty( $image_id ) ) {
-						$value = pods_image_url( $image_id, $size, 0, true );
+			// Default image field handlers.
+			$image_fields = array(
+				'image_attachment',
+				'image_attachment_url'
```

Comment:
* :no_entry_sign: Comma required after last value in array declaration
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: ce3f468253f817a8863db2be98078269</sub>